docs/index.md

```diff
@@ -1,119 +1 @@
-provider "artifactory" {
-  url = "${var.artifactory_url}/artifactory"
-  username = "${var.artifactory_username}"
-  password = "${var.artifactory_password}"
-}
-
-# Create a new repository
-resource "artifactory_local_repository" "pypi-libs" {
-  key             = "pypi-libs"
-  package_type    = "pypi"
-  repo_layout_ref = "simple-default"
-  description     = "A pypi repository for python packages"
-}
-```
-
-## Authentication
-The Artifactory provider supports multiple means of authentication. The following methods are supported:
-
-    * Basic Auth
-    * Bearer Token
-    * JFrog API Key Header
-
-### Basic Auth
-Basic auth may be used by adding a `username` and `password` field to the provider block
-Getting this value from the environment is supported with the `ARTIFACTORY_USERNAME` and `ARITFACTORY_PASSWORD` variable
-
-Usage:
-```hcl
-# Configure the Artifactory provider
-provider "artifactory" {
-  url = "artifactory.site.com/artifactory"
-  username = "myusername"
-  password = "mypassword"
-}
-```
-
-### Bearer Token
-Artifactory access tokens may be used via the Authorization header by providing the `access_token` field to the provider
-block. Getting this value from the environment is supported with the `ARTIFACTORY_ACCESS_TOKEN` variable
-
-Usage:
-```hcl
-# Configure the Artifactory provider
-provider "artifactory" {
-  url = "artifactory.site.com/artifactory"
-  access_token = "abc...xy"
-}
-```
-
-### JFrog API Key Header
-Artifactory API keys may be used via the `X-JFrog-Art-Api` header by providing the `api_key` field in the provider block.
-Getting this value from the environment is supported with the `ARTIFACTORY_API_KEY` variable
-
-Usage:
-```hcl
-# Configure the Artifactory provider
-provider "artifactory" {
-  url = "artifactory.site.com/artifactory"
-  api_key = "abc...xy"
-}
-```
-
-## Argument Reference
-
-The following arguments are supported:
-
-* `url` - (Required) URL of Artifactory. This can also be sourced from the `ARTIFACTORY_URL` environment variable.
-* `username` - (Optional) Username for basic auth. Requires `password` to be set. 
-    Conflicts with `api_key`, and `access_token`. This can also be sourced from the `ARTIFACTORY_USERNAME` environment variable.
-* `password` - (Optional) Password for basic auth. Requires `username` to be set. 
-    Conflicts with `api_key`, and `access_token`. This can also be sourced from the `ARTIFACTORY_PASSWORD` environment variable.
-* `api_key` - (Optional) API key for api auth. Uses `X-JFrog-Art-Api` header. 
-    Conflicts with `username`, `password`, and `access_token`. This can also be sourced from the `ARTIFACTORY_API_KEY` environment variable.
-* `access_token` - (Optional) API key for token auth. Uses `Authorization: Bearer` header. For xray functionality, this is the only auth method accepted
-    Conflicts with `username` and `password`, and `api_key`. This can also be sourced from the `ARTIFACTORY_ACCESS_TOKEN` environment variable.
+# JFrog Xray Provider
```

Comment:
You're going to have to fill this all in at some point

Reply:
For sure